arches_component_lab/src/arches_component_lab/utils.ts

```diff
@@ -1,68 +0,0 @@
-import { ALT_LABEL, PREF_LABEL } from "@/arches_component_lab/constants.ts";
```

Comment:
This is used in both controlled-lists and lingo. Is there a plan to move this to controlled-lists?

Reply:
Yep! No reason for it to live here